### PR TITLE
Create service and view to create a new book suggestion without authentication

### PR DESCRIPTION
**Pull request by fedecvz**
## Summary:
Create service and view to create a new book suggestion without authentication

## Trello card:
https://trello.com/c/FfIvGdOA/28-crear-vista-para-crear-book-suggestions
create_book_suggestion

### CODE REVIEW

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
-        book_suggestion.save!
-        render json: book_suggestion
+        create_new_book_suggestion
+      end
+
+      def create_without_auth
+        create_new_book_suggestion
       end
 
+      def new; end
```
Comment (juanisierra) [inline app/controllers/api/v1/book_suggestions_controller.rb]: why do you need to define a new?

File: app/controllers/api/v1/book_suggestions_controller.rb [hunk 2]
```diff
@@ -1,17 +1,29 @@
 module Api
   module V1
     class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_api_v1_user!, only: %i[new create_without_auth]
```
Comment (juanisierra) [inline app/controllers/api/v1/book_suggestions_controller.rb]: You can leave the create unauthenticated so you don't need to authenticate through jQuery

File: config/routes.rb [hunk 3]
```diff
@@ -5,7 +5,8 @@
     namespace :v1 do
       resources :books, only: [:show, :index]
       resources :rents, only: [:create, :index], path: 'users/:user_id/rents'
-      resources :book_suggestions, only: [:create]
+      resources :book_suggestions, only: [:create, :new]
+      post '/book_suggestions/new', to: 'book_suggestions#create_without_auth'
```
Comment (juanisierra) [inline config/routes.rb]: Use the default  create endpoint but unauthenticated

**Review by juanisierra — CHANGES_REQUESTED** [review]

File: app/controllers/api/v1/book_suggestions_controller.rb (hunk 1 above)
Comment (fedecvz) [reply app/controllers/api/v1/book_suggestions_controller.rb]: Because it is an offense to rubocop to do it in more than one line.

**Review by juanisierra — APPROVED** [review]

**Pull request merged**